lib/sonata/staking/executeChannelUnstake.ts

```diff
@@ -26,17 +27,15 @@ async function executeChannelUnstake({ channelId, amount, accessToken }: Channel
 
   const { channelAddress } = info;
 
-  const userStakeAmount = Math.abs(await getPoints(user.verifications, userEvent));
+  const userStakeAmount = -(await getPoints(user.verifications, userEvent));
```

Comment:
Could you please help me understand how the `-` here impacts the `-` on line 35?
- would it be easier to read if we remove the `-` on line `30` and negate the value on `35-36`???